src/timelines/preamble.js

```diff
@@ -1,17 +1,23 @@
-import experimentStart from '../trials/experimentStart'
-import startCode from '../trials/startCode'
-import userId from '../trials/userId'
+import { jsPsych } from 'jspsych-react'
+import { eventCodes, lang } from '../config/main'
 import holdUpMarker from '../trials/holdUpMarker'
-import { AT_HOME, IS_ELECTRON } from '../config/main'
-
+import {showMessage, userId} from '@brown-ccv/behavioral-task-trials'
+import { config, AT_HOME } from '../config/main'
+console.log(showMessage('html_button_response',undefined, config,lang.task.name, false, true, undefined, undefined, [lang.prompt.continue.button]))
 console.log('at_home', AT_HOME)
 console.log('env at home', process.env.REACT_APP_AT_HOME)
 const preamble = {
   type: 'html_keyboard_response',
   stimulus: '',
-  timeline: (AT_HOME || !IS_ELECTRON) ?
-    [experimentStart(), userId()] :
-    [experimentStart(), userId(), holdUpMarker(), startCode()]
+  timeline: (!config.USE_PHOTODIODE) ?
+    [showMessage('html_button_response',undefined, config,`<h1>${lang.task.name}</h1>`, false, true, undefined, undefined, [lang.prompt.continue.button]),
```

Comment:
will show message even not be an `html_button_esponse`? same with userId and `html_keyboard_response` - do those need to be passed in? What are the undefineds?

could this block of code be prettified so it's easier to read?